config/manager/manager.yaml

```diff
@@ -22,8 +22,6 @@ spec:
       labels:
         control-plane: controller-manager
     spec:
-      securityContext:
-        runAsNonRoot: true
```

Comment:
This was causing errors in the development environment and I noticed no other provider was setting this for their manager.yaml.

If we don't want to remove this no worries, I'll try to find a way with it `true`